packages/common/src/search/types/IHubCatalog.ts

```diff
@@ -203,4 +203,10 @@ export interface IGalleryDisplayConfig {
   showLinkButton?: boolean;
   linkButtonStyle?: "solid" | "outline" | "outline-fill" | "transparent";
   linkButtonText?: string;
+  sort?: string;
```

Comment:
I think instead of `string`, we should specify the types, e.g. ` "relevance" | "title" | "created" | "modified" `